Dockerfile.arm64

```diff
@@ -0,0 +1,23 @@
+# Build the manager binary
+FROM golang:1.13 as builder
```

Comment:
Let's use `golang:1.14` instead.

Reply:
Sounds good, I was simply not wanting to stray from `v1.13` which we were already using for amd64 builds. Should I update both?

Reply:
AMD64 should use `1.14` unless you aren't on latest `master`. If that's the case, please merge latest `master` :)

Reply:
:+1:

Reply:
```suggestion
FROM golang:1.14 as builder
```